e2e/cases/assets-retry/src/App.tsx

```diff
@@ -14,7 +14,7 @@ const App = () => {
         </ErrorBoundary>
       </div>
       <div style={{ border: '1px solid white', padding: 20 }}>
-        <ErrorBoundary elementId='"async-comp-test-error"'>
```

Comment:
React.ErrorBoundary is necessary in our e2e case, because we hope our async chunk error can be caught by errorBoundary

Retry will solve the white screen problem caused by `Chunk loading error` in some cases